scripts/cybershake/task_config.yaml

```diff
@@ -0,0 +1,5 @@
+EMOD3D: ALL
+HF: ALL
+BB: ALL
+IM_calc: ALL
+clean_up: ALL
```

Comment:
As people are probably going to use this a the base for their own config, the other tasks (get the str_value from each task in qcore.constants.ProcessType) should be listed with the keyword NONE.